README.md

```diff
@@ -45,25 +47,13 @@ There are some minor annoyances:
 1. Install the following software:
     - A text editor, like [Sublime](https://www.sublimetext.com/), which is what you'll use write the thesis.  
     - A LaTeX distribution (for example, [MacTeX](https://tug.org/mactex/) for Mac users).
-    - [Pandoc](http://johnmacfarlane.net/pandoc), for converting the Markdown to the output format of your choice.  You may also need to install [Pandoc cite-proc](http://pandoc.org/demo/example19/Extension-citations.html) to create the bibliography.
-    - Install @martisak's shortcaption module for Pandoc, with `pip install pandoc-shortcaption`
+    - [Pandoc](http://johnmacfarlane.net/pandoc), for converting the Markdown to the output format of your choice.  
+    - Pandoc plugins by running ```bash install.sh``` in the main direcotry
```

Comment:
Did you mean this bash script? I don't see it! Should the install instructions be added to the makefile instead? (e.g. with a `make install` recipe)

Reply:
Ah, that script didn't make it to a later release, since the Dockerfile made it redundant. I've added it back in. Having a `make install` recipe is an interesting idea. I'm going to put both in, as I don't want the lack of a shebang in the Makefile causing anyone headaches.